{"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"networkError","kind":"identifier"}],"languages":["swift"]}]}],"sections":[],"variants":[{"paths":["\/documentation\/identityorchestration\/tsidojourneyerror\/networkerror"],"traits":[{"interfaceLanguage":"swift"}]}],"abstract":[{"type":"text","text":"Could not connect to server, or server did not respond before timeout."}],"identifier":{"url":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError\/networkError","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration","doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError"]]},"metadata":{"role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"networkError","kind":"identifier"}],"title":"TSIdoJourneyError.networkError","symbolKind":"case","roleHeading":"Case","externalID":"s:21IdentityOrchestration17TSIdoJourneyErrorO07networkE0yA2CmF","modules":[{"name":"IdentityOrchestration"}]},"kind":"symbol","schemaVersion":{"major":0,"minor":3,"patch":0},"references":{"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration/TSIdoJourneyError":{"abstract":[{"type":"text","text":"The enum for the start Journey errors."}],"role":"symbol","title":"TSIdoJourneyError","type":"topic","kind":"symbol","url":"\/documentation\/identityorchestration\/tsidojourneyerror","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"TSIdoJourneyError"}],"navigatorTitle":[{"text":"TSIdoJourneyError","kind":"identifier"}],"identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError"},"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration/TSIdoJourneyError/networkError":{"url":"\/documentation\/identityorchestration\/tsidojourneyerror\/networkerror","kind":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"networkError"}],"type":"topic","identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError\/networkError","abstract":[{"type":"text","text":"Could not connect to server, or server did not respond before timeout."}],"role":"symbol","title":"TSIdoJourneyError.networkError"},"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration":{"type":"topic","role":"collection","title":"IdentityOrchestration","kind":"symbol","abstract":[],"url":"\/documentation\/identityorchestration","identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration"}}}